docs/build/group-chat.md

```diff
@@ -1,44 +1,199 @@
 ---
 sidebar_label: Group chat
 sidebar_position: 4
-description: Learn how to create, list, and manage group chats with XMTP
+description: Learn how to build group chat with XMTP
 ---
 
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
 # Build group chat with XMTP
 
-:::info JavaScript
+Secure group chat is an important part of every messaging app. In this guide, we cover the essentials of building group chat using XMTP, from the initial steps of ensuring that potential members have v3 identities and starting a new group chat to managing membership and synchronizing group chat details.
 
-At present, the JavaScript SDK lacks support for Group Chat functionalities. Nevertheless, for those looking to integrate backend features, the CLI provides a viable solution, as detailed in [this repository](https://github.com/xmtp/libxmtp/tree/main/examples/cli). For practical application, an example implementation is available on [Replit](https://replit.com/@neekolas/Groups-Nodejs-Client#src/index.ts). To explore group functionalities further, refer to the comprehensive [Token Gated Group Chat Tutorial](/docs/tutorials/token-gated-group-chat).
+XMTP group chat is based on the [MLS specification](https://www.rfc-editor.org/rfc/rfc9420.html) and can include anywhere from 1 to 400 members.
 
-:::
+To learn about group chat security and encryption, see [Group Chat](/docs/concepts/v3/group-chat.md) concepts.
+
+## Overview
+
+Here are some key points to understand before building group chat with XMTP.
+
+### Group chat keys work per app installation
+
+An app installation is registered to a user. Group chat messages a user sends using an app installation are encrypted so only that app installation can decrypt them. This is because keys are generated to work per app installation. App installations do not share the same keys.
+
+To learn more, see [Installations](/docs/concepts/v3/identity#installations).
+
```

Comment:
@nplasterer If you have time, I am wondering if I might get one last look from you on the topic below? 

I THINK I have it - but since the topic is so important - I'd love to get the details right. <3 Thank you!